sway-ir/tests/cse/cse3.ir

```diff
@@ -0,0 +1,29 @@
+// regex: ID=[[:alpha:]0-9]+
+// regex: VAR=v\d+
+
+script {
+    entry fn main(a: u64, b: u64) -> () {
+        entry(a: u64, b: u64):
+        // check: $(v5=$VAR) = add a, b
+        v5 = add a, b
+        v6 = const u64 0
+        br while(v6, v5)
+
+        while(v3: u64, v4: u64):
```

Comment:
`v4` is not being used anymore, right? Will this be removed later?

Reply:
Yes, `v4` is unused. No, our DCE doesn't remove unused arguments, but we could extend it to so.